client/blocks/plugins-update-manager/schedule-list.tsx

```diff
@@ -114,7 +116,9 @@ export const ScheduleList = ( props: Props ) => {
 						canCreateSchedules && (
 							<Text as="p">
 								<Icon className="icon-info" icon={ info } size={ 16 } />
-								The current feature implementation only allows to set up two schedules.
+								{ translate(
+									'The current feature implementation only allows to set up two schedules.'
```

Comment:
"Only" implies that we know 2 are not enough or very little. Can we just remove this entire thing?

Reply:
We can't remove the whole sentence, or we are not going to manage the user expectations properly.
We can say;
- The current feature implementation allows to set up of two schedules.

Reply:
Yeah, I agree that we should cover this case and provide additional info.

I'll take the option "The current feature implementation allows to set up two schedules."